test/services_manager_service_test.rb

```diff
@@ -563,6 +565,21 @@
     end
   end
 
+  describe "#status" do
+    let(:fake_cmd_result) { { "stdout" => "Fake status output" } }
+
+    before do
+      allow(Yast2::Systemctl).to receive(:execute).with(/status/).and_return(fake_cmd_result)
+    end
+
+    it "queries for the status of each service keyword" do
+      expect(Yast2::Systemctl).to receive(:execute).with(/cups\.service/)
```

Comment:
np: you do not check that it is calling `status`, just that it is doing *something* with them.

Reply:
Nice, good catch! Thank you